tests/pytests/pkg/conftest.py

```diff
@@ -370,10 +359,7 @@ def salt_master(salt_factories, install_salt, pkg_tests_account):
         scripts_dir.mkdir(exist_ok=True)
         salt_factories.scripts_dir = scripts_dir
         python_executable = install_salt.bin_dir / "Scripts" / "python.exe"
-        if install_salt.classic:
-            python_executable = install_salt.bin_dir / "python.exe"
-        if install_salt.relenv:
-            python_executable = install_salt.install_dir / "Scripts" / "python.exe"
+        python_executable = install_salt.install_dir / "Scripts" / "python.exe"
```

Comment:
This is re-assigning the `python_executable` variable from the line above with the previous value never being used.